include/na/NAGraphAlgorithms.hpp

```diff
@@ -152,7 +152,8 @@ class NAGraphAlgorithms {
    * @return A tuple of vectors containing the moveable vertices, the fixed
    * vertices mapped to their relative x-position in every time step.
    */
-  [[nodiscard]] static auto computeSequence(const InteractionGraph& g)
+  [[nodiscard]] static auto computeSequence(const InteractionGraph& g,
+                                            std::size_t             maxSites)
```

Comment:
The docstring of this method should be updated to reflect the additional parameter.